lib/rspec/graphql_matchers/have_a_return_field.rb

```diff
@@ -0,0 +1,16 @@
+module RSpec
+  module GraphqlMatchers
+    class HaveAReturnField < HaveAField
+      def initialize(expected_field_name)
+        super
+        @fields = :return_fields
+      end
+
+      def matches?(graph_object)
```

Comment:
Hah oops you're right 😅 Missed that in the last refactor commit. Since that's the case we could even remove these classes altogether and just pass in the `fields` as an initializer param.